Source/Core/RectangleGeometry.js

```diff
@@ -802,6 +802,7 @@ define([
         options.textureMatrix = textureMatrix;
         options.tangentRotationMatrix = tangentRotationMatrix;
         options.size = options.width * options.height;
+        options.stRotation = stRotation;
```

Comment:
Sorry to be dense, but where is that happening?  From what I can tell, only `constructRectangle` and `constructExtrudedRectangle` get called with this specific options object and neither of them use it.